code/modules/client/preferences_savefile.dm

```diff
@@ -309,7 +309,20 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	READ_FILE(S["randomise"],  randomise)
 	READ_FILE(S["persistent_scars"] , persistent_scars)
 
-	READ_FILE(S["loadout_list"] , loadout_list)//SKYRAT EDIT ADDITION
+	//SKYRAT EDIT ADDITION
+	READ_FILE(S["loadout_list"] , loadout_list)
```

Comment:
you know you can just `/datum/preferences/load_character` and `..()` right? you can make this modular

Reply:
Oh. True. I'll do that, thanks

Reply:
you guys do this a LOOOOT ive noticed this is just the first time ive felt the need to point it out